frontend/src/global_styles/content/work_packages/tabs/_tab_content.sass

```
@@ -27,20 +27,18 @@
 //++
 
 .op-tab-content
-  &--container
-    .op-files-tab--storage-info-box:not(:last-child)
-      margin-bottom: $spot-spacing-1
-
   &--tab-section
     margin-bottom: $spot-spacing-2
 
     &_no-header
-      .op-files-tab
-        &--file-upload
-          margin-top: 0
+      .op-files-tab--file-list .op-files-tab--file-list-item:first-child
+        margin-top: 0
+
+    .op-files-tab--storage-info-box:not(:last-child)
+      margin-bottom: $spot-spacing-1
 
-        &--file-list .op-files-tab--file-list-item:first-child
-          margin-top: 0
+    .op-files-tab--file-list-actions
+      margin-top: $spot-spacing-0-75
```

```suggestion
      margin-top: $spot-spacing-0_75
```

In sass, underscores and dashes are treated equally (which is why this code works), but we've tried to use dashes for word separation, while the underscore denotes the decimal dot.